controller/backup_controller.go

```diff
@@ -832,6 +854,14 @@ func (bc *BackupController) checkMonitor(backup *longhorn.Backup, volume *longho
 	}
 	if snapshot != nil {
 		if !snapshot.Status.ReadyToUse {
+			bc.creationRetryCounter.IncreaseCount(backup.Name)
+			if bc.creationRetryCounter.GetCount(backup.Name) >= maxCreationRetry {
+				backup.Status.Error = fmt.Sprintf(FailedWaitingForSnapshotMessage, backup.Spec.SnapshotName)
+				backup.Status.State = longhorn.BackupStateError
```

Comment:
If it is marked as Error, when will the volumeattachment be deleted?

Reply:
Yes, the `error` state is [a final state](https://github.com/longhorn/longhorn-manager/blob/6ab807644f15ec6c5941eeebe3a93780613e238b/controller/backup_controller.go#L999-L1004) so VA ticket will be deleted here https://github.com/longhorn/longhorn-manager/blob/6ab807644f15ec6c5941eeebe3a93780613e238b/controller/backup_controller.go#L380-L382

Reply:
Thank you.